### PR TITLE
Get output from SSH commands

### PR DESCRIPTION
**Pull request by mnowaksuse**
And remove Xen HVM emulator setup which is not needed.

### CODE REVIEW

**Comment by okurz** [comment]
LGTM

File: consoles/sshVirtsh.pm [hunk 1]
```diff
@@ -330,6 +317,27 @@ sub add_disk {
     return;
 }
 
+# In list context returns pair ($stdout, $stderr). In void (and scalar)
+# context just logs stdout and stderr, returns nothing.
+sub get_ssh_outputs {
+    my $chan = shift;
```
Comment (coolo) [inline consoles/sshVirtsh.pm]: please don't use shift for parameters. Look around in this file

**Comment by coolo** [comment]
I'm a bit puzzled by the plural. I consider stderr and stdout part of the one output, it's just two channels of it.

**Comment by mnowaksuse** [comment]
Sub is now in singular.

**Pull request merged**